### ISSUE TITLE
Members of the collaborative groups

### ISSUE THREAD

**Issue by askarlupka** [EDRN/P5#323]
## 🤔 Tell Us About the Feature

I'm frustrated when I am trying to find members of different collaborative groups both the PIs and the NCI affiliates for each of the research groups but that information is behind a log-in wall. 

## 🎇 What Solution You'd Like

I want the to the collaborative groups on this page: https://edrn.nci.nih.gov/about-edrn/groups/ to go to a page similar to this: https://edrn.nci.nih.gov/about-edrn/groups/collaboration-and-publication-subcommittee/ which has an overview of what it the group is and its members. Then on that page have the link to access the private information for that group. 

## 〽️ Alternative Ideas

I would just like some where intuitive to have which members are affiliated with which groups. Especially NCI program officers so that external PIs know who is the contact person at NCI for that group.

**Comment by nutjob4life** [comment]
@hoodriverheather check out this issue from @askarlupka.

I think we could easily accommodate this by making the organ collaborative groups (and any other private groups) public, but keeping their sub-pages (web pages and committee events) private to those groups. That way, any visitor could come along to say the Lung group, see the officers, scientists, chair, co-chair, and members.

We could include a message like "Are you a member of this group? Logging in may give access to private group-specific documents and events."

What do you think?

**Comment by askarlupka** [comment]
I love that idea!! I was confused given the log-in prompt because it allows me to log into the three different locations and I didn't know which one contained the information that was supposed to be on that page. So maybe also adding text above the log-in prompt maybe giving a very general description of what information regarding that group could be found by going through the different log-in prompts. Happy to create a new issue for this specific request in the future though because it is different

**Comment by nutjob4life** [comment]
Hi @askarlupka, I think tracking it in the comments on this single issue is fine 😇

I'll let @hoodriverheather chime in just in case she has some other ideas.

Code freeze for 6.9 is a week from today but if this idea passes muster I think we can fit this in easily enough.

**Comment by hoodriverheather** [comment]
This is a great idea! Thanks @askarlupka . @nutjob4life for the login, would it be possible when a user is requesting a page from the Public Portal to direct them to a specific public portal login page?

**Comment by hoodriverheather** [comment]
@nutjob4life I think we do need to keep the Network Consulting Team page completely private. We could confirm with DMCC or NCI :)

**Comment by nutjob4life** [comment]
@hoodriverheather not sure what's meant by "specific public portal login page".

We have the "one true login page" and it automatically redirects back to whatever page you were looking at before, so if we were to add a login link to each committee/collaborative group/working group/etc., then after logging in you'd end up back at that committee/collaborative group/working group/etc..

Hope this helps!

**Comment by hoodriverheather** [comment]
@nutjob4life I was just wondering if it could direct you specifically to the EDRN login prompts.
<img width="683" alt="Screenshot 2024-02-01 at 10 23 28 AM" src="https://github.com/EDRN/P5/assets/58526347/919e6ed1-f536-4ea0-a4e0-fa4f804cef1f">

That way if a user is coming from the public portal and needs to login, they don't have to figure out which of the 3 options to select. IDK. This has always been confusing and hard to explain. :)

**Comment by nutjob4life** [comment]
Ahhh, I see what you mean. Yeah, you know you want to log into the portal and _not_ LabCAS _nor_ the "secure" site, so those other options are just a bunch of visual noise. 

Yes, I think we can do something like that.

**Comment by nutjob4life** [comment]
Argh, I thought this would be a small change but it's actually enormous.

- The portal currently uses Wagtail "page view restrictions" on the various groups in order to keep those groups that should be private, private. 
- To show the members of groups to the public, I need to remove the page view restrictions.
- But this means that the various agendas, PowerPoints, and other files would become public.
- I have a feeling they'd prefer some/all of these to remain private.
- We can offer a feature where each group has two "folders", one for private things for members-only and one for public things they want to share with the public—but this is extra work
- But this would also apply to events…do we want events to be public or can they all be private? Or will groups want a mix of public/private events?—which is a lot of extra work
- Or can we assume that everything a committee/working group/collaborative group is to remain private to that group but just the membership of the group is public; in which case visiting such a group would show the name of the group and its members but a button like <kbd>Click here to access this group's workspace which includes its events, files, agendas, etc.</kbd>—which would simplify things!—and in this case anything they'd want to "publish" would go somewhere else on the portal?

Looking for input from both of you!

**Comment by askarlupka** [comment]
Interesting, I think these questions are worth bringing up to the groups.
I'll take a look at these points next week and circulate them around to the
NCI side.

**Comment by nutjob4life** [comment]
@hoodriverheather yeah, that's what I'm describing in my 7th bullet item in [this comment](https://github.com/EDRN/P5/issues/323#issuecomment-1924675843), and I think it's a nice balance. 

You get the "landing page" with the name of the group and its members. And it makes all the powerpoints and PDFs and agendas private under a clearly labeled "workspace" on the landing page—and that requires a login.

**Comment by hoodriverheather** [comment]
@nutjob4life I have lost exactly what I'm supposed to test on this. Can you please remind me? Sorry!

**Comment by nutjob4life** [comment]
@hoodriverheather this is slated for milestone 6.12. You don't need to do anything on it—but thanks in advance! 😅 

[View the open 6.11 issues here](https://github.com/EDRN/P5/milestone/34)
